paddle/trainer/tests/test_CompareSparse.cpp

```diff
@@ -146,12 +146,12 @@ TEST(compareSparse, remote_cpu) {
 TEST(compareSparse, cpu10_local_vs_remote) {
   FLAGS_local = 1;  // disable remote sparse update in parameter config
   std::vector<ParameterPtr> localParameters =
-      trainerOnePassTest(configFile1, true, 10);
+      trainerOnePassTest(configFile1, true, 2);
```

Comment:
When PaddlePaddle linked with Openblas on MAC OS X,  run unit test triggers "35: BLAS : Program is Terminated. Because you tried to allocate too many memory regions. ", I guess the problem is from openblas.